### PR TITLE
Trivy Support

### PR DESCRIPTION
**Pull request by shanice-skylight**
# DEVOPS PULL REQUEST

### Related Issue
[#99](https://app.zenhub.com/workspaces/dibbs-cloud-enablement-659dde695f06770a91327bda/issues/gh/cdcgov/dibbs-cloud/99)

### Changes Proposed
- This Pull request adds Trivy to the repository, it will help identify potential vulnerabilities that might get introduced with each PR.
- This scans the code in the `terraform` folder in the branch of the PR, if deemed necessary the scope of the scan can be modified as well as artifacts such as images can be included. 

### Additional Information
- This update will run Trivy in Repo Mode: This tells Trivy to scan the repository.
- It will scan for Vulnerabilities, Misconfigurations, and Secrets

###Testing

### CODE REVIEW

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.20.0
+        with:
+          scan-type: 'fs'
+          scan-ref: 'terraforms/'
```
Comment (rin-skylight) [inline .github/workflows/trivy.yml:24]: This doesn't appear to be the correct directory.

File: .github/workflows/trivy.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    name: trivy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.20.0
```
Comment (rin-skylight) [inline .github/workflows/trivy.yml:21]: It looks like the current version is `0.23.0`.

**Review by rin-skylight — CHANGES_REQUESTED** [review]

File: .github/workflows/trivy.yml (hunk 1 above)
Comment (shanice-skylight) [reply .github/workflows/trivy.yml]: Fixed directory name

File: .github/workflows/trivy.yml (hunk 2 above)
Comment (shanice-skylight) [reply .github/workflows/trivy.yml]: Updated package version.

**Review by alismx — APPROVED** [review]
:rocket:

**Pull request merged**